### PR TITLE
Add callback URL type to REST client

### PR DESCRIPTION
**Pull request by romanstrobl**
The missing URL type was causing an exception on server when running tests.

### CODE REVIEW

File: powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/PowerAuthClient.java [hunk 1]
```diff
@@ -782,12 +782,13 @@ VaultUnlockResponse unlockVault(String activationId, String applicationKey, Stri
      *
      * @param applicationId Application ID.
      * @param name          Callback URL display name.
+     * @param type          Callback type.
```
Comment (petrdvorak) [inline powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/PowerAuthClient.java:785]: I will introduce an enum for this...

File: powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthRestClient.java [hunk 2]
```diff
@@ -691,10 +691,11 @@ public CreateCallbackUrlResponse createCallbackUrl(CreateCallbackUrlRequest requ
     }
 
     @Override
-    public CreateCallbackUrlResponse createCallbackUrl(Long applicationId, String name, String callbackUrl, List<String> attributes) throws PowerAuthClientException {
+    public CreateCallbackUrlResponse createCallbackUrl(Long applicationId, String name, String type, String callbackUrl, List<String> attributes) throws PowerAuthClientException {
```
Comment (petrdvorak) [inline powerauth-rest-client-spring/src/main/java/com/wultra/security/powerauth/rest/client/PowerAuthRestClient.java:694]: I will introduce enum for this.

**Comment by petrdvorak** [comment]
The changes I am making will make better sense when the SOAP interface is fully gone.

**Review by petrdvorak — APPROVED** [review]
Looks OK

**Comment by romanstrobl** [comment]
Ok, thanks.

**Pull request merged**